app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragment.java

```diff
@@ -388,6 +385,35 @@ private void setupPOIListListener() {
         });
     }
 
+    private void removeAllRoomsFromMap(){
+        for (Marker marker : roomMarkers) marker.remove();
+        roomMarkers.clear();
+    }
+
+    private void removeAllPOIFromMap(){
+        for (Marker marker : poiMarkers) marker.remove();
+        poiMarkers.clear();
+    }
```

Comment:
Cann't we create one generic for both methods? Since, it seems that they are serving exactly the  same purpose.

Reply:
ok